Task_TheGridProblem/index.css

```diff
@@ -1 +1,79 @@
-/* Start coding below for GRID PROBLEM TASK */
+.container{
+  display:grid;
+  grid-template-rows: 200px 200px 200px;
+  grid-template-columns: 200px 200px 200px;
+  background-color: #a2d2ff;
+  padding: 30px;
+  gap:3px;
+  }
+  .grid-item1{
+  text-align:center;
+  border: 1px solid rgba(17, 17, 17, 0.8);
+  padding: 18px;
+  font-size: 28px;
+  background-color: rgba(255, 255, 255, 0.8);
+  }
+  .grid-item2 {
+  text-align:center;
+  border: 1px solid rgba(17, 17, 17, 0.8);
+  padding: 18px;
+  font-size: 28px;
+  background-color: rgba(255, 255, 255, 0.8);
+  }
+  .grid-item3 {
+  text-align:center;
+  border: 1px solid rgba(17, 17, 17, 0.8);
+  padding: 18px;
+  font-size: 28px;
+  background-color: rgba(255, 255, 255, 0.8);
+  }
+  .grid-item4 {
+  text-align:center;
+  border: 1px solid rgba(17, 17, 17, 0.8);
+  padding: 18px;
+  font-size: 28px;
+  background-color: rgba(255, 255, 255, 0.8);
+  }
+  .grid-item5 {
+  text-align:center;
+  border: 1px solid rgba(17, 17, 17, 0.8);
+  padding: 18px;
+  font-size: 28px;
+  background-color: rgba(255, 255, 255, 0.8);
+  }
+  .grid-item6 {
+  text-align:center;
+  border: 1px solid rgba(17, 17, 17, 0.8);
+  padding: 18px;
+  font-size: 28px;
+  background-color: rgba(255, 255, 255, 0.8);
+  }
+  .grid-item7 {
+  text-align:center;
+  grid-column-start:1 ;
```

Comment:
The 'grid-column-end' property should be set to '4' instead of '3' to span across all columns.